dashboard/test/ui/features/star_labs/applab/scenarios.feature

```diff
@@ -49,36 +49,39 @@ Feature: App Lab Scenarios
     And I wait until element "#divApplab > .screen > div#text_area1" is visible
     Then element "div#text_area1" has html "Line 1<div>Line 2</div><div><br></div><div>Line3</div>"
 
-  Scenario: Change event works in text input and text area
+  Scenario: Change event works in text input
     Given I switch to design mode
     And I drag a TEXT_INPUT into the app
-    And I drag a TEXT_AREA into the app
     And I switch to code mode
     And I ensure droplet is in text mode
     And I append text to droplet "onEvent('text_input1', 'change', function(event) {\n"
     And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_input1'));\n"
-    And I append text to droplet "});\n\n"
-    And I append text to droplet "onEvent('text_area1', 'change', function(event) {\n"
-    And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_area1'));\n"
     And I append text to droplet "});"
 
     # in text input, blur produces a change event
     When I press "runButton"
-    And I wait until element "#divApplab > .screen > div#text_area1" is visible
-    And I press keys "123" for element "#text_input1"
+    And I wait until element ".screen > input" is visible
+    And I press keys "123" for element ".screen > input"
```

Comment:
This could still be called `#text_input1`, right? Any particular reason for the renaming? And it looks like this wasn't explicitly waiting on the text input value before (but it wasn't causing problems?).

Reply:
Yes, it can still be called `"text_input1"`. 
At some point when the test was not passing, I changed the ref to `".screen > input"`. But it was another part of the scenario that was buggy and I didn't change it back!
I'll go ahead and use `"text_input1"` for consistency. Thanks!